composer.json

```diff
@@ -32,7 +32,7 @@
         "bin-dir": "bin/"
     },
     "require": {
-        "php": ">=5.4",
+        "php": ">=5.5",
```

Comment:
Because I use [generators](https://github.com/K-Phoen/rulerz/pull/21/files#diff-f81ca9c9f07519e024ec9edb6a3c0920R31) (and PHP 5.4 isn't maintained anymore…)